app/src/components/learningJourney/staff/StaffLearningJourney.jsx

```diff
@@ -36,7 +37,9 @@ function StaffLearningJourney(staffId) {
         learningJourneysReturnedFromBackend[i].isJobActive = result[i].isActive;
       }
 
-      setLearningJourneys(learningJourneysReturnedFromBackend);
+      setLearningJourneys(mockLearningJourneys)
+
+      // setLearningJourneys(learningJourneysReturnedFromBackend);
```

Comment:
can you revert this, mocks shouldnt be used remotely anymore since the api endpoint is up and in use

Reply:
Done